TF_Service_Utilities/ServiceUITray/ServiceUITray/Program.cs

```diff
@@ -47,62 +49,39 @@ public class ServiceUITray : ApplicationContext
 
         public ServiceUITray()
         {
-            if (File.Exists(APPLICATION_PATH))
-            {
-                InitializeTray(true);
-            }
-            else
+            touchFreeMenuItem = new MenuItem("TouchFree Application", LaunchApp);
+
+            trayIcon = new NotifyIcon()
             {
-                InitializeTray(false);
-            }
+                Icon = Properties.Resources.IconActive,
+                ContextMenu = new ContextMenu(new MenuItem[] {
+                    touchFreeMenuItem,
+                    new MenuItem("-"),
+                    new MenuItem("Service Settings", Settings),
```

Comment:
Unless this has changed recently, I believe this should just say 'Settings' rather than 'Service Settings'?